.gitignore

```diff
@@ -31,3 +31,9 @@ Carthage
 # `pod install` in .travis.yml
 #
 # Pods/
+
+# Fastlane
+report.xml
+
+# Example
+Example/Pods/Target Support Files/
```

Comment:
Please change this to `Example/Pods` and add new line.

Reply:
@mkj-is Done.